data/sql_updates/create_candidate_detail_vw.sql

```diff
@@ -0,0 +1,56 @@
+drop view if exists ofec_candidate_detail_vw;
+create view ofec_candidate_detail_vw as
+select
+    max(dimcand.cand_sk) as candidate_key,
```

Comment:
Anything in the GROUP BY should not be using an aggregate function. Remove `max()`.